src/common/CMakeLists.txt

```diff
@@ -1,15 +1,14 @@
 add_subdirectory(binaries_op)
-add_subdirectory(bzip2_op)
```

Comment:
Why are you removing this library? Please add this line again, it is being used by `data_provider` and `version_op`.